helpers/compile/plugins/fill-plugin/fillPlugin.ts

```diff
@@ -191,14 +191,14 @@ const fillPlugin = (
       process: {
         inject: path.join(__dirname, 'fillers', 'process.ts'),
       },
-      global: {
```

Comment:
Not 100% this should be removed yet.

Reply:
(I'll re-add as needed one I get to writing the ecosystem tests for this)

Reply:
I think this needs to be restored, I will make it more resilient in the finalization PR.